src/Workspaces/CoreTest/EditorConfigStorageLocation/EditorConfigStorageLocationTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles;
+using Microsoft.CodeAnalysis.Options;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests.EditorConfig.StorageLocation
+{
+    public class EditorConfigStorageLocationTests
+    {
+        [Fact]
+        public static void TestEmptyDictionaryReturnFalse()
```

Comment:
Can you add NamingStyle in some form to the test name? Can you mention that the cases where this returns true are covered in another test class, with a pointer to that place?